chiron/tests/test_testsystems.py

```diff
@@ -0,0 +1,19 @@
+from chiron.integrator import LangevinIntegrator
+
+
+def test_HO():
+    from openmm.unit import kelvin
+    from chiron.potential import HarmonicOscillatorPotential
+    from openmmtools.testsystems import HarmonicOscillator
+
+    ho = HarmonicOscillator()
+    harmonic_potential = HarmonicOscillatorPotential(ho.K, ho.positions, ho.U0)
+    integrator = LangevinIntegrator(harmonic_potential, ho.topology)
+    integrator.run(ho.positions, temperature=300 * kelvin, n_steps=1000)
+
+    class State:
+        def __init__(self, temperature):
+            self.temperature = temperature
+
+    ho.get_potential_expectation(State(300 * kelvin))
+    ho.get_potential_standard_deviation(State(300 * kelvin))
```

Comment:
I'm not sure this openmmtools function does the correct thing, looks like it just returns the energy, not standard deviation: 

https://github.com/choderalab/openmmtools/blob/0bbef98a404997b1503c6d6a7777bc0157fb6426/openmmtools/testsystems.py#L818